src/bio/smd/SConscript

```diff
@@ -6,10 +6,11 @@ def scons():
     env.AppendUnique(LIBPATH=[Dir('.')])
 
     denv = env.Clone()
-
+    denv.Append(CPPDEFINES=['-DDAOS_PMEM_BUILD'])
```

Comment:
this line is not needed.

Reply:
Encountered some failures without these as btree_class.c also has a ifdef